pkg/daemon/controller/controller.go

```diff
@@ -477,16 +537,57 @@ func (c *Controller) iptablesSyncLoop() {
 				iptablesManager := c.getIPtablesManager(subnet.Spec.Range.Version)
 				iptablesManager.RecordSubnet(cidr, ramav1.GetNetworkType(network) == ramav1.NetworkTypeOverlay)
 			}
+		}
+
+		if daemonfeature.MultiClusterEnabled() {
```

Comment:
Maybe move this `if daemonfeature.MultiClusterEnabled() {}` block into `if overlayExist{}` block, because "have a overlay network" is a prerequisite for multicluster.